tools/lsmcli/cmdline.py

```diff
@@ -1822,7 +1822,9 @@ def local_disk_list(self, args):
                     info_dict[key] = func_dict[key](disk_path)
                 except LsmError as lsm_err:
                     if lsm_err.code != ErrorNumber.NO_SUPPORT:
-                        raise
+                        print("WARN: %s('%s'): %d %s" %
```

Comment:
The only bad thing about doing this is if anyone is scripting lsmcli this will break them.  I wonder if we shouldn't print this on stderr instead of stdout?  At least then a user could sort out what they want to parse.

Reply:
Changed to stderr.